app/helpers/users_helper.rb

```diff
@@ -13,34 +13,27 @@ def gravatar_for_email(email, options = {size: 256, class: 'h-6 w-6 rounded-full
     image_tag(gravatar_url, alt: email, class: options[:class])
   end
 
-  def self.sunday_ratio_for(user)
-    return nil unless user.lid?
-
-    date = user.lid_since
-    drinks = Event.where(attendance: true).where("date > ?", date).where("deadline < ?", Date.today)
+  def self.drink_ratio_for(user)
+    drinks = Event.where(attendance: true).where('date > ?', Date.today - 1.year).where('deadline < ?', Date.today)
```

Comment:
Date.today - 1.year is wel naar voor nieuwe mensen, die kunnen na een jaar pas op de 100% komen. Misschien date wel op user.lid_since zetten en dan als > 1 jaar deze op dat jaar zetten.

Reply:
https://github.com/DispuutHamers/webapp/pull/460